packages/electron-builder/src/options/winOptions.ts

```diff
@@ -347,4 +357,19 @@ export interface AppXOptions {
    * Describes the contents of the package. The Name attribute is case-sensitive. Corresponds to [Identity.Name](https://msdn.microsoft.com/en-us/library/windows/apps/br211441.aspx).
    */
   readonly identityName?: string | null
+
+  /**
+   * Runs makepri on the folder before running makeappx.
+   */
+  readonly makePri?: boolean | null
+
+  /**
+   * Specify a custom assets folder to use for the assets.
+   */
+  readonly assetsFolder?: string | null
```

Comment:
This dir should be equals by default to some convenient location, e.g. appxAssets. So, user can simply put files without explicit conf.